ibm_mq/assets/configuration/spec.yaml

```diff
@@ -141,6 +141,16 @@ files:
         value:
           example: false
           type: boolean
+      - name: queue_manager_timezone
+        description: |
+          Timezone of the queue manager if it is different from the host system timezone. Valid values are a
+          time zone name (IANA, or, on Windows, Windows keys), location of a tzfile(5) zoneinfo file or
+          TZ variable style time zone specifier. If `localtime`, queue_manager_timezone will be set to host's
+          local system timezone.
```

Comment:
```suggestion
          Time zone of the queue manager if it differs from the host system's time zone. Valid values include a 
          time zone name (IANA or Windows keys), a location of a tzfile(5) zoneinfo file, or a 
          TZ variable style time zone specifier. If `localtime`, `queue_manager_timezone` is set to the host's
          local system time zone.
```

Reply:
Thanks for reviewing! I changed the language for this section. Would you be able to take another look?